tests/js/server/aql/aql-functions.js

```diff
@@ -2905,375 +1960,124 @@ function ahuacatlFunctionsTestSuite () {
     
     testFlatten4 : function () {
       var input = [ 1, [ 2, [ 3, [ 4, [ 5, [ 6, [ 7, [ 8, [ 9 ] ] ] ] ] ] ] ] ];
-      var buildQuery = function (nr, input) {
-        switch (nr) {
-          case 0:
-            return `RETURN FLATTEN(@value, ${input})`;
-          case 1:
-            return `RETURN NOOPT(FLATTEN(@value, ${input}))`;
-          default:
-          assertTrue(false, "Undefined state");
-        }
-      };
-
-      for (var i = 0; i < 2; ++i) {
-        var q = buildQuery(i, 1);
-        var actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, [ 3, [ 4, [ 5, [ 6, [ 7, [ 8, [ 9 ] ] ] ] ] ] ] ] ], actual, q);
-
-        q = buildQuery(i, 2);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, [ 4, [ 5, [ 6, [ 7, [ 8, [ 9 ] ] ] ] ] ] ] ], actual, q);
-
-        q = buildQuery(i, 3);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, [ 5, [ 6, [ 7, [ 8, [ 9 ] ] ] ] ] ] ], actual);
-
-        q = buildQuery(i, 4);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, 5, [ 6, [ 7, [ 8, [ 9 ] ] ] ] ] ], actual);
-
-        q = buildQuery(i, 5);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, 5, 6, [ 7, [ 8, [ 9 ] ] ] ] ], actual);
-
-        q = buildQuery(i, 6);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, 5, 6, 7, [ 8, [ 9 ] ] ] ], actual);
-
-        q = buildQuery(i, 7);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, 5, 6, 7, 8, [ 9 ] ] ], actual);
-
-        q = buildQuery(i, 8);
-        actual = getQueryResults(q, { value: input });
-        assertEqual([ [ 1, 2, 3, 4, 5, 6, 7, 8, 9 ] ], actual);
-      }
-    },
-
-////////////////////////////////////////////////////////////////////////////////
-/// @brief test min function
-////////////////////////////////////////////////////////////////////////////////
-    
-    testMin1 : function () {
```

Comment:
Is this test and some of the following tests just deleted? They don't seem to use NOOPT, and I don't replacements for them.
But I may have simply overlooked it given the size of the PR.

Reply:
It still exists: https://github.com/arangodb/arangodb/blob/935964279127ef0a6db6b2063736e4fdb1ef4afb/tests/js/server/aql/aql-functions.js#L1992-L1996

Reply:
Ok.